packages/jest-environment-webdriver/modules/WebDriverEnvironment.js

```diff
@@ -6,14 +6,17 @@ class WebDriverEnvironment extends NodeEnvironment {
     super(config);
     const options = config.testEnvironmentOptions || {};
     this.browserName = options.browser || 'chrome';
+    this.usingServer = options.usingServer || null;
```

Comment:
I'd call the option `seleniumAddress` so it would have some context when written in Jest config. What do you think?

Reply:
I like it )